projects/js-packages/charts/src/components/line-chart/line-chart.tsx

```diff
@@ -102,6 +122,7 @@ const LineChart: FC< LineChartProps > = ( {
 	legendOrientation = 'horizontal',
 	withGradientFill = false,
 	smoothing = true,
+	curveType = 'linear',
```

Comment:
We could potentially get rid of `smoothing` option it feels. We need to change Jetpack Stats accordingly tho.